keyboards/gggw/crosses/lib/crosses.c

```diff
@@ -0,0 +1,225 @@
+        scroll_acc_h -= (int16_t)scroll_acc_h;
+        scroll_acc_v -= (int16_t)scroll_acc_v;
+
+        // Clear the X and Y values of the mouse report
+        mouse_report.x = 0;
+        mouse_report.y = 0;
+    }
+
+    return mouse_report;
+}
+
+#endif /* RIGHT || LEFT */
+
+/***********************************************************************
+ * Dual Trackball Config
+ ***********************************************************************/
+
+#ifdef POINTING_DEVICE_COMBINED
+
+report_mouse_t pointing_device_task_combined_user(report_mouse_t left_report, report_mouse_t right_report) {
+    // Calculate and accumulate scroll values based on mouse movement and divisors
+    scroll_acc_h += (float)left_report.x / SCROLL_DIVISOR_H;
+    scroll_acc_v += (float)left_report.y / SCROLL_DIVISOR_V;
+
+    // Assign integer parts of accumulated scroll values to the mouse report
+    left_report.h = (int16_t)scroll_acc_h;
+    left_report.v = (int16_t)scroll_acc_v;
+
+    // Update accumulated scroll values by subtracting the integer parts
+    scroll_acc_h -= (int16_t)scroll_acc_h;
+    scroll_acc_v -= (int16_t)scroll_acc_v;
+
+    // Clear the X and Y values of the mouse report
+    left_report.x = 0;
+    left_report.y = 0;
+
+    return pointing_device_combine_reports(left_report, right_report);
+}
+
+#endif /* ifdef POINTING_DEVICE_COMBINED */
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+#if defined(POINTING_DEVICE_RIGHT) || defined(POINTING_DEVICE_LEFT)
+    if (get_highest_layer(state) != 5) {
+        set_scrolling = false;
+    }
+#endif /* ifndef POINTING_DEVICE_* */
+
+    return state;
+}
+
+void secondary_sync_handler(uint8_t in_buflen, const void* in_data, uint8_t out_buflen, void* out_data) {
+    if (in_buflen == sizeof(global_user_config)) {
+        memcpy(&global_user_config, in_data, sizeof(global_user_config));
+    }
+}
+
+void housekeeping_task_kb(void) {
+    if (!is_keyboard_master()) { return; }
+
+    static global_user_config_t last_config = {0};
+    static uint32_t last_sync = 0;
+    bool should_sync = false;
+
+    if (memcmp(&global_user_config, &last_config, sizeof(global_user_config))) {
+        should_sync = true;
+        memcpy(&last_config, &global_user_config, sizeof(global_user_config));
+    }
+
+    should_sync =  (timer_elapsed32(last_sync) > 500);
+
+    if (!should_sync) { return; }
+
+    if (transaction_rpc_send(CROSSES_SECONDARY_SYNC_ID, sizeof(global_user_config), &global_user_config)) {
+        last_sync = timer_read32();
+    }
+}
+
+void keyboard_post_init_user(void) {
+    global_user_config.raw = eeconfig_read_user();
+    transaction_register_rpc(CROSSES_SECONDARY_SYNC_ID, secondary_sync_handler);
+
+#if defined(POINTING_DEVICE_COMBINED) || defined(POINTING_DEVICE_RIGHT) || defined(POINTING_DEVICE_LEFT)
+    update_pointer_cpi(&global_user_config);
+#endif /* ifdef POINTING_DEVICE_* */
+
+    write_config_to_eeprom(&global_user_config);
```

Comment:
```suggestion
    write_config_to_eeprom(&global_user_config);
    
    keyboard_post_init_user();
```

Reply:
For my edification. Should L217 also change to `eeconfig_read_kb`?